examples/sampleview/core/sampledesigner/DesignerHelper.cpp

```diff
@@ -145,43 +139,25 @@ int DesignerHelper::nanometerToScreen(double nanometer)
     return result;
 }
 
-QRectF DesignerHelper::getDefaultBoundingRect(const QString &name)
+QRectF DesignerHelper::getDefaultBoundingRect(const std::string& name)
```

Comment:
At some point we have to seat and decide, how we proceed here. We need to take care of hdpi monitors in a right way straight from the beginning.

Reply:
I guess it can be done as it is described in [qt documentation](https://doc.qt.io/qt-5/highdpi.html#high-dpi-support-in-qt). Will keep it in mind.